### PR TITLE
4 aug

### PR DESCRIPTION
**Pull request by MarianBastiurea**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -80,20 +80,24 @@
...
             <artifactId>tomcat-embed-jasper</artifactId>
             <scope>provided</scope> <!--default for IntelliJ -->
         </dependency>
 
+
         <dependency>
-            <groupId>org.glassfish.web</groupId>
-            <artifactId>jakarta.servlet.jsp.jstl</artifactId>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-rest-hal-explorer</artifactId>
```
Comment (andreiursudev) [inline pom.xml:101]: Why replace this dependency and not put it next to it?

File: pom.xml [hunk 2]
```diff
@@ -80,20 +80,24 @@
...
         </dependency>
+
+
         <dependency>
-            <groupId>jakarta.servlet.jsp.jstl</groupId>
-            <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-hateoas</artifactId>
```
Comment (andreiursudev) [inline pom.xml:109]: Why replace this dependency and not put it next to it?

File: src/main/resources/schema.sql [hunk 3]
```diff
@@ -1,8 +1,10 @@
-create table course
```
Comment (andreiursudev) [inline src/main/resources/schema.sql:1]: Remove both schema.sql and data.sql files.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/resources/schema.sql (hunk 3 above)
Comment (MarianBastiurea) [reply src/main/resources/schema.sql:1]: done

File: pom.xml (hunk 1 above)
Comment (MarianBastiurea) [reply pom.xml:101]: done

File: pom.xml (hunk 2 above)
Comment (MarianBastiurea) [reply pom.xml:109]: done

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**